### ISSUE TITLE
Error running code

### ISSUE THREAD

**Issue by fanlala** [JuanDuGit/DH3D#8]
0%|                                                    |0/2073[00:00<?,?it/s] 2021-05-20 10:48:44.249488: E tensorflow/stream_executor/cuda/cuda_blas.cc:647] failed to run cuBLAS routine cublasSgemmBatched: CUBLAS_STATUS_EXECUTION_FAILED
2021-05-20 10:48:44.249528: E tensorflow/stream_executor/cuda/cuda_blas.cc:2505] Internal: failed BLAS call, see log for details
2021-05-20 10:48:44.887998: I tensorflow/stream_executor/stream.cc:4817] stream 0x55c2525213f0 did not memzero GPU location; source: 0x7f1da27fab50
2021-05-20 10:48:44.888031: I tensorflow/stream_executor/stream.cc:4817] stream 0x55c2525213f0 did not memzero GPU location; source: 0x7f1da27fab70
2021-05-20 10:48:44.888071: E tensorflow/stream_executor/cuda/cuda_dnn.cc:2833] failed to enqueue forward batch normalization on stream: CUDNN_STATUS_EXECUTION_FAILED

InternalError (see above for traceback): Blas xGEMMBatched launch failed : a.shape=[10,512,3], b.shape=[10,3,3], m=512, n=3, k=3, batch_size=10
	 [[Node: MatMul = BatchMatMul[T=DT_FLOAT, adj_x=false, adj_y=false, _device="/job:localhost/replica:0/task:0/device:GPU:0"](split, _arg_R_0_0/_17)]]

I used cuda9.0 and tensorflow1.9, ubuntu18.04，Can you take a look at it for me?

**Comment by JuanDuGit** [comment]
It seems like this error is GPU dependent. You should check and make sure you have compatible cudatoolkit & cudnn & tensorflow to run on your gpu card. As I can't reproduce the error, I am afraid I could not be of more help on this issue.
